service/src/main/kotlin/fi/espoo/evaka/serviceneednew/NewServiceNeedController.kt

```diff
@@ -56,6 +58,8 @@ class NewServiceNeedController(
                 shiftCare = body.shiftCare,
                 confirmedAt = HelsinkiDateTime.now()
             )
+                .let { id -> tx.getNewServiceNeedChildRange(id) }
```

Comment:
We already have all the needed data so querying it separately seems unnecessary.

Reply:
we dont have childId, just placementId

Reply:
Oh yeah, forgot about that 😩